src/pages/experiments/[experimentId]/plots-and-tables/frequency/index.jsx

```diff
@@ -101,7 +105,23 @@ const FrequencyPlotPage = ({ experimentId }) => {
   if (!config) {
     return <Skeleton />;
   }
+  const formatCSVData = (plotData) => {
+    const newCsvData = [];
+
+    louvainClusters[0].children.forEach((cluster) => {
+      const entriesForCluster = plotData.filter((entry) => entry.yCellSetKey === cluster.key);
+      const cellSetName = cellSets.properties[cluster.key].name;
+      const newEntry = { 'Louvain Cluster': cellSetName };
```

Comment:
Likewise, you might want to remove the use of the term `louvain`, in case we use other cell sets (e.g. custom, or even leiden later on)